scanners/nikto/examples/demo-bodgeit/findings.yaml

```diff
@@ -7,119 +7,1729 @@
     "name": "The anti-clickjacking X-Frame-Options header is not present.",
```

Comment:
cut this down to 5 findings. These examples are rendered on the docs website and 5 should be sufficient. Also do this for the other findings in the examples folder.